### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by moetayuko**

### CODE REVIEW

File: install.sh
Comment (clsty) [inline install.sh:1]: I don't understand, why add `/usr/local/lib/` things? I can't find all of the same files inside `/usr/lib` after installing those local PKGBUILDs, e.g. `/usr/lib/Gvc-1.0.typelib`.

Comment (moetayuko) [reply install.sh:1]: they're leftovers of `install-ags`

**Review by clsty — APPROVED** [review]
That's fine for me.

Though some other things related to AGS needs to be reviewed by @end-4 .

**Review by end-4 — COMMENTED** [review]
AGS stuff are alright

**Comment by end-4** [comment]
Good
I used gojq because I wanted my bash workspace indicator script to be fast, but now I don't use bash for that anymore so it's nice to use jq instead of gojq to reduce an AUR dependency

**Comment by moetayuko** [comment]
> Good I used gojq because I wanted my bash workspace indicator script to be fast, but now I don't use bash for that anymore so it's nice to use jq instead of gojq to reduce an AUR dependency

you might want to switch to jaq for speed, which is available in Arch extra as well. https://github.com/01mf02/jaq?tab=readme-ov-file#performance

**Comment by end-4** [comment]
iirc i had issues with jaq how it's args were a bit different... i could be wrong and can't be bothered to investigate at this point

**Comment by moetayuko** [comment]
@clsty I just noticed that `illogical-impulse-basic` was not rebuilt by the script because pkgver was not updated. We need to either bump pkgver of these meta-packages when updating, or pass `-f` to makepkg. Which is preferred in your sense?

**Comment by clsty** [comment]
> @clsty I just noticed that `illogical-impulse-basic` was not rebuilt by the script because pkgver was not updated. We need to either bump pkgver of these meta-packages when updating, or pass `-f` to makepkg. Which is preferred in your sense?

Sorry for late response. The former one sounds better to me, though I don't know how to achieve that. PR is welcome.

**Comment by moetayuko** [comment]
> > @clsty I just noticed that `illogical-impulse-basic` was not rebuilt by the script because pkgver was not updated. We need to either bump pkgver of these meta-packages when updating, or pass `-f` to makepkg. Which is preferred in your sense?
> 
> Sorry for late response. The former one sounds better to me, though I don't know how to achieve that. PR is welcome.

Increasing pkgrel every time after updating pkgbuild should be sufficient.

**Pull request merged**